.github/workflows/infrastructure-cleardown.yaml

```diff
@@ -0,0 +1,103 @@
+name: Cleardown Infrastructure
+
+on:
+  workflow_call:
+    inputs:
+      github_environment:
+        description: "Defines the Github environment in which to pull environment variables from"
+        required: true
+        type: string
+      branch_name:
+        description: "Name of the branch used to derive tf version and workspace"
+        required: true
+        default: 'not-set'
+        type: string
+
+jobs:
+  echo-workflow-inputs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: echo workflow inputs
+        run: |
+          echo "github environment: ${{ inputs.github_environment }}"
+          echo "branch_name: ${{ inputs.branch_name }}"
+
+  minMax:
+    name: "Derive TF Version"
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+
+    outputs:
+      reponame: ${{ steps.get_metadata.outputs.reponame }}
```

Comment:
Is this used ... just not clear why we are outputting repo name from a job that is about deriving hte tf version
I'd probably also change name of job to something more descriptive than minMax (which is probably a legacy from when we validated terraform against a min and a max version rather than just the max)